docs/wiki.md

```diff
@@ -134,10 +134,10 @@ console.log(suggestion); //Returns 'Batman'
 
 ### random()
 
-Returns a random wiki page in any of the available formats. Formats can be `summary`, `title`, `related`, `mobile-sections` or `mobile-sections-lead`. Defaults to summary by default.
+Returns a random wiki page in any of the available formats. Formats can be `summary`, `title`, `html`, `related`, `mobile-sections` or `mobile-sections-lead`. Defaults to summary.
```

Comment:
We need to remove `html` from here as well.

Reply:
Fixed in latest commit